source/NuGet.config

```diff
@@ -4,6 +4,7 @@
     <!-- remove any machine-wide sources with <clear/> -->
     <clear />
     <add key="AdaptiveCards-Public" value="https://pkgs.dev.azure.com/ms/AdaptiveCards/_packaging/AdaptiveCards-Public/nuget/v3/index.json" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
Maybe the cppwinrt 210913.7 should get pushed into the AC feed? I'm nervous about adding the additional source here, but having it definitely made life easier.

Reply:
Agreed, I'm following up on my end on what our process is supposed to be here for picking up the new version.

Reply:
Ok, I've pushed the updated packages and this is now working against the AC feed.